packages/youtube-video-element/youtube-video-element.js

```diff
@@ -11,14 +11,20 @@ const PLAYLIST_MATCH_SRC =
   /(?:youtu\.be\/|youtube(?:-nocookie)?\.com\/.*?[?&]list=)([\w_-]+)/;
 
 function getTemplateHTML(attrs, props = {}) {
+  const { referrerpolicy, ...config } = props.config ?? {};
+
   const iframeAttrs = {
-    src: serializeIframeUrl(attrs, props),
+    src: serializeIframeUrl(attrs, { ...props, config }),
```

Comment:
```suggestion
    src: serializeIframeUrl(attrs, props),
```

Reply:
I'm not sure I follow this, all that logic is done in the function serializeIframeUrl

please revert